addons/mail/static/src/js/composer.js

```diff
@@ -329,17 +331,35 @@ var MentionManager = Widget.extend({
 
     // Cursor position and selection utils
     _get_selection_positions: function () {
```

Comment:
should be in private section, should be renamed _getSelectionPositions, should have docstring